tests/python/frontend/pytorch/test_forward.py

```diff
@@ -4079,5 +4078,45 @@ def test_fn(m, v):
     verify_model(test_fn, [torch.randn(3, 8), torch.randn(8)])
 
 
+def test_grid_sample():
+    class Grid_sample_zeros(Module):
+        def forward(self, x, y):
+            return torch.nn.functional.grid_sample(
+                input=x, grid=y, mode="bilinear", padding_mode="zeros", align_corners=True
+            )
+
+    class Grid_sample_border(Module):
+        def forward(self, x, y):
+            return torch.nn.functional.grid_sample(
+                input=x, grid=y, mode="bilinear", padding_mode="border", align_corners=True
+            )
+
+    data = torch.rand([4, 4, 16, 32]).float()
+    grid = torch.rand([4, 8, 8, 2]).float()
+    verify_model(Grid_sample_zeros(), input_data=[data, grid])
+    verify_model(Grid_sample_border(), input_data=[data, grid])
+
+
+def test_list_tuple():
+    """test compilation error for a Python list followed by a prim::TupleConstruct."""
```

Comment:
Can you give me more details on what this test is about? I don't see a relevant change in `pytorch.py`.

Reply:
There was a bug in PyTorch frontend on list of tuples.  And the bug is now already fixed upstream.  I removed my fix in this patch but decided to keep the unit test.